pom.xml

```diff
@@ -65,6 +66,7 @@ under the License.
         <assertj.version>3.21.0</assertj.version>
         <archunit.version>0.22.0</archunit.version>
         <testcontainers.version>1.17.2</testcontainers.version>
+        <!--    TODO: Remove Mockito -->
```

Comment:
+1 to this

Reply:
Punted this to a followup to simplify this change